app/controllers/api/subscriptions_controller.rb

```diff
@@ -0,0 +1,40 @@
+class Api::SubscriptionsController < ApplicationController
+  before_action :authenticate_user!
+
+  def create
+      customer_id = get_customer(params[:stripeToken])
+      subscription = Stripe::Subscription.create({customer: customer_id, plan: 'DreamTimePlan'})
+    
+      test_env?(customer_id, subscription)
+      status = Stripe::Invoice.retrieve(subscription.latest_invoice).paid
+
+    if status
+      current_user.update_attribute(:role, 'subscriber')
+      render json: { message: 'You are now a subscriber'}, status: 201
```

Comment:
Please don't use the same uncreative messages that I used in the demo 🙃

Reply:
Changed it to Something went wrong. It's is very creative :)

Reply:
I meant the success message.